src/bench/random.cpp

```diff
@@ -378,12 +391,13 @@ bench_thread(PriorityQueue *pq,
                              settings.size - thread_id * slice_size : slice_size;
     for (int i = 0; i < initial_size; i++) {
         uint32_t elem = keygen.next();
+        uint32_t value = keygen.next();
```

Comment:
My mistake again. I think I added a separate value to test something and then I must have forgotten to change it back.